example-apps/kitchen-sink-example/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "angular2": "2.0.0-beta.1",
+    "angular2": "2.0.0-beta.3",
```

Comment:
zone.js dependency needs to be upgraded to 0.5.11 as there is a warning when building and running it that is related